src/app/services/student-book-services/student-book.service.ts

```diff
@@ -0,0 +1,98 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class StudentBookService {
+  private URL = 'https://fierce-shore-32592.herokuapp.com/diaries';
```

Comment:
wait for @tarasIV `s PR to fix this accordingly

Reply:
@dmitryy90  Done!

Reply:
must be '/diaries'

Reply:
Yeah,I forgot to commit it

Reply:
@dmitryy90  Done!